splice_machine/src/test/java/com/splicemachine/derby/impl/sql/execute/operations/SparkExplainIT.java

```diff
@@ -34,6 +36,7 @@
 import static com.splicemachine.test_tools.Rows.row;
 import static com.splicemachine.test_tools.Rows.rows;
 
+@Category(HBaseTest.class)
```

Comment:
This category makes the IT ineligible for remote mode. Is it justified? If yes, please update remote_it/pom.xml.

Reply:
Thanks @OlegMazurov The reason I put this test in the HBaseTest category is to prevent it from running on the mem profile. There are two reasons: 1) sparkexplain under mem profile does not really return spark explain, it returns the control execution plan. 2) the plan from control path is a bit different from the spark plan. 
However, I don't intend to make the IT ineligible for remote mode. Is there a way that I can make it eligible for remote test but still not triggered for mem profile?

Reply:
I was wrong assuming that the category brings dependencies on HBase. I don't think we should exclude the test from _remote_it_.